File: src/codes/excessive-branching/php/ksrc.php
<?php

function generateNotification(string $userType, string $message): string {
  if ($userType === "guest") {
    return "Hello Guest! {$message}";
  } elseif ($userType === "member") {
    return "Hi Member! {$message}";
  } elseif ($userType === "admin") {
    return "Dear Admin,\n{$message}";
  } elseif ($userType === "moderator") {
    return "Moderator Notice: {$message}";
  } else {
    return "Hello User! {$message}";
  }
}

File: src/codes/excessive-branching/py/ksrc.py
def generate_notification(user_type: str, message: str) -> str:
    if user_type == "guest":
        return f"Hello Guest! {message}"
    elif user_type == "member":
        return f"Hi Member! {message}"
    elif user_type == "admin":
        return f"Dear Admin,\n{message}"
    elif user_type == "moderator":
        return f"Moderator Notice: {message}"
    else:
        return f"Hello User! {message}"
